jobrunner/lib/database.py

```diff
@@ -19,6 +19,62 @@
 
 CONNECTION_CACHE = threading.local()
 
+TABLE_SCHEMAS = {
```

Comment:
:+1: Using inline Python rather than an external file is a good move, as is splitting it per table. I feel like this deserves a separate `schema.py` file though, rather than bundling it in `database.py` which has already got quite a lot going on.

Reply:
OK, will do

Reply:
What do you think about embedding the schema in the dataclass definition?

e.g.

class Job:
   __tablename__ = "job"
   __tableschema__ = """
       CREATE TABLE job (
       ...
   """

Reply:
I did a thing for auto registering.